app/pages/external/ConsignmentAdditionalInformationTypePage.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.external
+
+import models.Index
+import pages.ReadOnlyPage
+import pages.sections.external.ConsignmentAdditionalInformationSection
+import play.api.libs.json.JsPath
+
+case class ConsignmentAdditionalInformationTypePage(index: Index) extends ReadOnlyPage[String] {
+
+  override def path: JsPath = ConsignmentAdditionalInformationSection(index).path \ "type" \ "code"
```

Comment:
Need to check this path is correct as it doesn't look like it has been built yet. Waiting on https://jira.tools.tax.service.gov.uk/browse/CTCP-5144